### ISSUE TITLE
[Enhancement]: On aws_s3_bucket resource, disable setting default object ownership ACL to ObjectWriter

### ISSUE THREAD

**Issue by Nacymus** [hashicorp/terraform-provider-aws#28513]
### Description

When an S3 bucket is created, an ACL of type object ownership is set to object writer. There is no argument in the resource allowing to avoid this ACL being set.  With a resource aws_s3_bucket_ownership_controls it is possible to force the ACL to another value: either BucketOwnerEnforced or BucketOwnerPreferred, but no way of disabling it.

### Affected Resource(s) and/or Data Source(s)

aws_s3_bucket

### Would you like to implement a fix?

None

**Comment by justinretzolk** [comment]
Hey @Nacymus 👋 Based on my read of [this AWS document](https://docs.aws.amazon.com/AmazonS3/latest/userguide/about-object-ownership.html), it's seems it's expected for S3 buckets' default Object Ownership to be set to object writer, and that the `aws_s3_bucket_ownership_controls` resource be used to manipulate it.

**Comment by Nacymus** [comment]
Yes! but the tf resource doesn't allow to disble that default setting.

**Comment by justinretzolk** [comment]
Hey @Nacymus 👋 Thanks for following up! Can you give me a better idea of what you're looking to achieve? I'm not sure I'm following what you're looking for the resource to accomplish, but if the resource isn't behaving as expected, we certainly want to look at that.
